src/basic-samples/GuaranteedPublisher.js

```diff
@@ -0,0 +1,196 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Solace Web Messaging API for JavaScript
+ * Publishing Guaranteed messages on a Topic tutorial - Guaranteed publisher
+ * Demonstrates sending persistent messages on a topic
+ */
+
+/*jslint es6 browser devel:true*/
+/*global solace*/
+
+var GuaranteedPublisher = function (solaceModule, topicName) {
+    'use strict';
+    var publisher = {};
+    publisher.session = null;
+    publisher.topicName = topicName;
+
+    // Logger
+    publisher.log = function (line) {
+        var now = new Date();
+        var time = [('0' + now.getHours()).slice(-2), ('0' + now.getMinutes()).slice(-2),
+            ('0' + now.getSeconds()).slice(-2)];
+        var timestamp = '[' + time.join(':') + '] ';
+        console.log(timestamp + line);
+    };
+
+    publisher.log('\n*** publisher to topic "' + publisher.topicName + '/*" is ready to connect ***');
```

Comment:
we should remove `/*` in the console log

Reply:
And can we move this output log to 57 instead after a successful connection

Reply:
Done